### DISCUSSION TITLE
Experience the menu with the latest updates

### DISCUSSION THREAD

**Discussion by D-Borkan** [YimMenu/YimMenu#2458] [General]
i test it the menu for 3 day ago and no ban for yet ( from this update #2430 and above )
also i test it many features

so i dont think the menu is detect for now
but maybe feature or some feature is detect but i dont know if that is cause

- also methods of taking money beyond the permissible limit It may be a main reason for the ban ( like special cargo or nightclub ..... etc ) with regard of transaction error

**Comment by Rabtos** [comment]
Is this the reason (potential detection) why the menu has no downloadable .dll release build since ~2 weeks?

**Reply by D-Borkan** [reply]
yeah the download from release is disable ( with regard of detect )

**Reply by yubie-re** [reply]
Building your own DLL is it's own sort of defense against R*'s AC. Only takes a few minutes... so why take the risk of using public builds?

**Comment by lonelybud** [comment]
i have question for persist vehicle feature. It spawns the car and run bunch of natives to apply mods in a short matter of time. The original vehicle spawn also spawns the vehicle (without applying any mods on it). Is it like detectable? Other menus are still doing it, and they are undetected. Just wondering there is a slight chance of detection? @maybegreat48 @yubie-re

**Reply by maybegreat48** [reply]
No
